packages/mattermost-redux/src/actions/integrations.ts

```diff
@@ -293,6 +293,20 @@ export function getOAuthApps(page = 0, perPage: number = General.PAGE_SIZE_DEFAU
     });
 }
 
+export function getAppsOAuthAppsIDs(): ActionFunc {
+    return bindClientFunc({
+        clientFunc: Client4.getAppsOAuthAppsIDs,
+        onSuccess: [IntegrationTypes.RECEIVED_APPS_OAUTH_APP_IDS],
```

Comment:
Do we need the arrays around the `onSuccess` values? It looks inconsistent throughout the file, but it caught me off guard at first glance here.

Reply:
No. It is not needed. Nevertheless, for consistency, I would prefer to keep it as a list.

Reply:
The array is left over from when we always used to have a _SUCCESS action dispatched for every request in addition to something like a POST_RECEIVED action. I think it might still be used in some cases, but we could definitely consider adding a non-array form in the future.